src/main/resources/assets/js/app/issue/view/IssueStatusSelector.ts

```diff
@@ -1,15 +1,20 @@
 import {IssueStatus, IssueStatusFormatter} from '../IssueStatus';
 import TabMenuItem = api.ui.tab.TabMenuItem;
 import TabMenu = api.ui.tab.TabMenu;
+import DivEl = api.dom.DivEl;
+import i18n = api.util.i18n;
 
-type IssueOption = { value: IssueStatus, name: string };
+interface IssueOption {
+    value: IssueStatus;
+    name: string;
+}
 
 export class IssueStatusSelector
     extends TabMenu {
 
     private static OPTIONS: IssueOption[] = [
-        {value: IssueStatus.OPEN, name: 'Open'},
-        {value: IssueStatus.CLOSED, name: 'Closed'}
+        {value: IssueStatus.OPEN, name: 'open'},
+        {value: IssueStatus.CLOSED, name: 'closed'}
     ];
 
```

Comment:
Here we should also drop `OPTIONS` (and `IssueOption` interface) and work directly with `IssueStatus` enum (since we always use all statuses).

Reply:
I agree. I completely refactored the IssueStatus usage in IssueStatusSelector and removed the IssueOption.